datadog_operator/metadata.csv

```diff
@@ -1,2 +1,69 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+datadog.operator.admission_controller.feature.enabled,gauge,,,,`1` if the Admission Controller feature is enabled,0,datadog_operator,admission_controller_enabled,
+datadog.operator.agent.deployment.success,gauge,,,,`1` if the desired number of Agent replicas equals the number of available Agent pods. `0` otherwise.,0,datadog_operator,agent_deployment_success,
+datadog.operator.cluster_checks.feature.enabled,gauge,,,,`1` if the Cluster checks feature is enabled,0,datadog_operator,cluster_checks_enabled,
+datadog.operator.clusteragent.deployment.success,gauge,,,,`1` if the desired number of Cluster Agent replicas equals the number of available Cluster Agent pods. `0` otherwise.,0,datadog_operator,dca_deployment_success,
+datadog.operator.clusterchecksrunner.deployment.success,gauge,,,,`1` if the desired number of Cluster Check Runner replicas equals the number of available Cluster Check Runner pods. `0` otherwise.,0,datadog_operator,ccr_deployment_success,
+datadog.operator.controller_runtime_active_workers,gauge,,,,Number of currently used workers per controller,0,datadog_operator,cr_active_workers,
```

Comment:
we have a `worker` unit

The list can be found [here](https://github.com/DataDog/integrations-core/blob/c45614423f51f2c8337b99140359de0eef6f98f4/ddev/src/ddev/cli/validate/metadata_utils.py#L32)

Reply:
Addressed by https://github.com/DataDog/integrations-core/pull/15655/commits/facfe66276ae175e6b0fb0244f78b2fb21d0677f